src/app/models/workspace-definition.ts

```diff
@@ -89,5 +89,4 @@ export interface DataFeatures extends CommonFeatures {
     };
 }
 
-export interface GalleryFeatures extends CommonFeatures {
-}
+export type GalleryFeatures = CommonFeatures;
```

Comment:
I didn't realize we had so many empty interfaces. Why is it better for it to be changed this way?